.github/workflows/go.yml

```diff
@@ -8,6 +8,21 @@ on:
   workflow_dispatch:
 
 jobs:
+  golangci:
+    name: lint
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v4
+      - uses: actions/setup-go@v5
+        with:
+          go-version: '1.21'
+          cache: false
+      - name: golangci-lint
+        uses: golangci/golangci-lint-action@v4
+        with:
+          version: v1.54
+          args: --timeout=3m
```

Comment:
The linter will only flag copyright date issues if the file has been changed more recently than the copyright year in the file. The GitHub checkout action will not download the entire git history by default unless you specify `fetch-depth: 0` https://github.com/hyperledger/firefly-ui/releases/tag/v1.3.0 If you do not include this field, and you were to inspect the file system that git checked out, every file would have an edit/creation date that matched the date/time of the commit that it checked out. Thus the linter thinks every file was changed in the current year.

I think because of this, your job here is flagging many false negatives. I think the existing golangci job that runs as part of the Makefile should be sufficient. And the existing GitHub actions that we have checkout the full history to avoid this problem.

Reply:
@nguyer Got it.
I removed golangci-lint from github actions. It might be useful to merge it. if user touches this files he/she will need to fix this issues. It is not a big deal but still can be annoying. I goes through changes and marked file where issue is different then wrong year.